src/palace/manager/api/axis.py

```diff
@@ -262,7 +262,7 @@ def authorization_headers(self) -> dict[str, str]:
         )
         return dict(Authorization="Basic " + authorization_b64)
 
-    def _run_self_tests(self, _db: Session) -> Generator[SelfTestResult, None, None]:
+    def _run_self_tests(self, _db: Session) -> Generator[SelfTestResult]:
```

Comment:
Type defaults! Assuming this came from pyupgrade.

Reply:
It did! I was happy to see this as well!